src/api/webhooks.js

```diff
@@ -44,9 +44,9 @@ export class WebhooksApi extends BaseApi {
    * @param  {AuthCredentials} auth
    * @return {APIResponse}
    */
-  list(auth) {
+  list() {
```

Comment:
Perhaps this belongs as  a TODO, but `validateAuth` could also be moved to base. Specific instances of BaseApi should then simply specify an array of allowed auth mechanisms eg `['jwt', 'appToken']`